tests/test_black.py

```diff
@@ -1679,6 +1679,19 @@ def test_reformat_one_with_stdin_and_existing_path(self) -> None:
             # __BLACK_STDIN_FILENAME__ should have been stripped
             report.done.assert_called_with(expected, black.Changed.YES)
 
+    def test_reformat_one_with_stdin_empty(self) -> None:
+        """Check that black doesn't fail for empty stdin (issue/#2337)"""
```

Comment:
The docstring may be a bit redundant, because the "doesn't fail" could be communicated in the test name, which also is displayed when the tests are run. But take it or leave it.

Reply:
Exactly. Thank you!

```suggestion
```